### PR TITLE
HIP-540

### PR DESCRIPTION
**Pull request by stefan-stefanooov**
Update official Hedera documentation after the implementation of HIP-540

**Related issue(s)**:
https://github.com/hashgraph/hedera-services/issues/12270

Fixes #
https://github.com/hashgraph/hedera-docs/issues/635

### CODE REVIEW

File: sdks-and-apis/sdks/token-service/update-a-token.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Update a token
 
-A transaction that updates the properties of an existing token. The admin key must sign this transaction to update any of the token properties. The admin key can update exisitng keys, but cannot add new keys if they were not set during the creation of the token. If no value is given for a field, that field is left unchanged.
+A transaction that updates the properties of an existing token. The admin key must sign this transaction to update any of the token properties. With one exeption. All secondary keys can sing a transaction to change themselves. The admin key can update exisitng keys, but cannot add new keys if they were not set during the creation of the token. If no value is given for a field, that field is left unchanged.
```
Comment (petreze) [inline sdks-and-apis/sdks/token-service/update-a-token.md]: minor typos: exception*, sign*

File: sdks-and-apis/sdks/token-service/update-a-token.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
 
 For an immutable token (that is, a token created without an admin key), only the expiry may be updated. Setting any other field, in that case, will cause the transaction status to resolve to `TOKEN_IS_IMMUTABlE`.
```
Comment (petreze) [inline sdks-and-apis/sdks/token-service/update-a-token.md]: After HIP-540, a token is mutable and can be changed even if its created without an admin key! After the HIP, if a `TokenUpdateTx` is performed we will keep returning `TOKEN_IS_IMMUTABLE` when it tries to change non-key properties or the expiry without an admin key
Change comment according to the specification of the HIP

File: sdks-and-apis/sdks/token-service/update-a-token.md [hunk 3]
```diff
@@ -23,7 +23,9 @@ For an immutable token (that is, a token created without an admin key), only the
 
 #### Transaction Signing Requirements
 
-* Admin key is required to sign to update any token properties.
+* Admin key is required to sign to update any token properties. (exept the secondary keys)
```
Comment (petreze) [inline sdks-and-apis/sdks/token-service/update-a-token.md]: except for the expiry and all low priority keys

**Review by petreze — COMMENTED** [review]
_All secondary keys_ can be replaced with explicitly naming what those keys are (wipe key, kyc key, etc.)
_applicable in all occurrences_

File: sdks-and-apis/sdks/token-service/update-a-token.md [hunk 4]
```diff
@@ -23,7 +23,9 @@ For an immutable token (that is, a token created without an admin key), only the
 
 #### Transaction Signing Requirements
 
-* Admin key is required to sign to update any token properties.
+* Admin key is required to sign to update any token properties. (except for the expiry and all low priority keys)
+* A secondary key can update itself. Meaning it can sing a transaction that changes itself.
```
Comment (petreze) [inline sdks-and-apis/sdks/token-service/update-a-token.md]: change to sign*

File: sdks-and-apis/sdks/token-service/update-a-token.md [hunk 5]
```diff
@@ -23,7 +23,9 @@ For an immutable token (that is, a token created without an admin key), only the
 
 #### Transaction Signing Requirements
 
-* Admin key is required to sign to update any token properties.
+* Admin key is required to sign to update any token properties. (except for the expiry and all low priority keys)
+* A secondary key can update itself. Meaning it can sing a transaction that changes itself. 
+    Example: Wipe Key can sing a transaction that changes only the Wipe Key
```
Comment (petreze) [inline sdks-and-apis/sdks/token-service/update-a-token.md]: sign instead of sing

**Review by petreze — APPROVED** [review]

**Review by petreze — APPROVED** [review]

**Review by theekrystallee — APPROVED** [review]
Thanks for the PR! LGTM

**Pull request merged**